lldb/tools/lldb-dap/Handler/CompileUnitsRequestHandler.cpp

```diff
@@ -10,6 +10,7 @@
 #include "EventHelper.h"
 #include "JSONUtils.h"
 #include "RequestHandler.h"
+#include "llvm/ADT/StringRef.h"
```

Comment:
I suspect this line is not required as `GetString` was already used and returns a `StringRef`?